x-pack/auditbeat/module/system/process/process.go

```diff
@@ -376,6 +380,13 @@ func (ms *MetricSet) processEvent(process *Process, eventType string, action eve
 		event.RootFields.Put("user.group.name", process.Group.Name)
 	}
 
+	if len(process.CapEffective) > 0 {
```

Comment:
```suggestion
	if len(process.CapEffective) != 0 {
```
(`cap` and `len` are spec'd to return positive `int` values)
also below